rollup/plugins/bundlePlugins.js

```diff
@@ -108,7 +108,6 @@ export function makeTerserPlugin() {
           '_initStorage',
           '_support',
           // TODO: Get rid of these once we use the SDK to send replay events
-          '_prepareEvent', // replay uses client._prepareEvent
           '_metadata', // replay uses client.getOptions()._metadata
```

Comment:
Side-note: As discussed, I'll take a look if we can get rid of the `_metadata` entry here